tnc/data_handler.py

```diff
@@ -267,7 +267,7 @@ def process_data(self, bytes_out, freedv, bytes_per_frame):
                 structlog.get_logger("structlog").debug("PING RECEIVED....")
                 # = self.get_frequency_offset(freedv)
                 # we need to fix this later
-                frequency_offset = 0
+                frequency_offset = "0"
```

Comment:
Mh, I declared "frequency_offset" as string in "received_ping". Not sure why I did this, so it might not break things. However, I would suggest we should possibly remove this entire feature "frequency offset" from the code. Its causing too mich confusion when enabled.